models/believer/believer.sdf

```diff
@@ -0,0 +1,693 @@
+<sdf version='1.6'>
```

Comment:
Not matching the one on the config file.

Reply:
Fixed!